tests/s3/copy_object/test_copy_object_tag.py

```diff
@@ -0,0 +1,1104 @@
+        self.log.info("Step 5: Copy-object with destination tag set same as source object"
+                      "in the different bucket")
+        status, response = self.s3_test_obj.copy_object(self.bucket_name1,
+                                                        self.object_name1,
+                                                        self.bucket_name2,
+                                                        self.object_name2,
+                                                        TaggingDirective='REPLACE',
+                                                        Tagging=tag_str)
+        assert_utils.assert_true(status, response)
+        copy_etag = response['CopyObjectResult']['ETag']
+        self.log.info("Step 6: Retrieving tag of a destination object %s", self.object_name2)
+        copy_resp = self.tag_obj.get_object_tags(self.bucket_name2, self.object_name2)
+        assert_utils.assert_true(copy_resp[0], copy_resp[1])
+        self.log.info("Retrieved tag of an object")
+        self.log.info("Step 7: Compare tagset of source and destination object")
+        assert_utils.assert_not_equal(copy_resp[1][0]["Key"], put_resp[1][0]["Key"],
+                                      put_resp[1][0]["Key"])
+        assert_utils.assert_not_equal(copy_resp[1][0]["Value"], put_resp[1][0]["Value"],
+                                      put_resp[1][0]["Value"])
+        self.log.info("Verify destination tag set match with assigned tag set in copy object")
+        assert_utils.assert_equal(copy_resp[1][0]["Key"], self.key_dest, self.key_dest)
+        assert_utils.assert_equal(copy_resp[1][0]["Value"], self.value_dest, self.value_dest)
+        self.log.info("Step 7: Compared and verified tags of source and destination object"
+                      "are different and do not match")
+        self.log.info("Step 8: Verification of etag and metadata for data integrity check")
+        self.copy_obj_di_check(self.bucket_name1, self.object_name1, self.bucket_name2,
+                               self.object_name2, put_etag=put_etag, copy_etag=copy_etag)
+        self.log.info("Step 8: Verification of etag and metadata for data integrity check")
+        self.log.info("ENDED: copy-object with destination tag set same as source object"
+                      "(--tagging-directive=REPLACE)")
+
+    @pytest.mark.parallel
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-44813")
+    def test_44813(self):
+        """
+        TEST-44813 :Test simple copy-object with source object having max number of tags(10)
+        and parameter (--tagging-directive=COPY)
+        """
+        self.log.info("STARTED: Simple copy-object with source object having max number of"
+                      "tags(10) and parameter (--tagging-directive=COPY)")
+        self.log.info("Copy-object with source object having max number of tags(10)"
```

Comment:
this statement is not needed as its captured in above line

Reply:
Explained